discord/message.py

```diff
@@ -721,6 +725,12 @@ def __init__(
         except KeyError:
             self.interaction = None
 
+        self.thread: Optional[Thread]
+        try:
+            self.thread = Thread(guild=self.guild, state=state, data=data['thread'])
```

Comment:
Not sure about this..
Too many API calls.

Reply:
I could be mistaken but I don't think it's an API call. Just see discord.Thread source code.